core/controllers/editor_test.py

```diff
@@ -762,39 +779,6 @@ def test_record_user_saw_tutorial(self):
         self.logout()
 
 
-class StateAnswerStatisticsHandlerTests(test_utils.GenericTestBase):
```

Comment:
Why is this deleted? 

Consider preemptively justifying any deletions in a comment to the review thread, otherwise reviewers will just ask that question during reviews and that will slow things down.

Reply:
StateAnswerStatisticsHandlerTests was added in PR https://github.com/oppia/oppia/pull/6679. That's why I deleted this duplicate test class